pages/_app.js

```diff
@@ -1,33 +1,98 @@
 import GlobalStyle from "@/styles";
-import projects from "@/lib/projects.js";
-import { Saira } from "next/font/google";
+import initialProjects from "@/lib/projects.js";
 import useLocalStorageState from "use-local-storage-state";
-
-const saira = Saira({ subsets: ["latin"] });
+import Layout from "@/components/Layout";
+import { nanoid } from "nanoid";
 
 export default function App({ Component, pageProps }) {
-  const [newProjects, setNewProjects] = useLocalStorageState("newProjects", {
-    defaultValue: projects,
+  const [projects, setProjects] = useLocalStorageState("projects", {
+    defaultValue: initialProjects,
   });
 
+  function handleAddProject(newProject) {
+    setProjects([newProject, ...projects]);
+  }
+
+  function handleDeleteProject(id) {
+    setProjects(projects.filter((project) => project.id !== id));
+  }
+
   function handleUpdateProject(updatedProject) {
-    setNewProjects(
-      newProjects.map((project) =>
+    console.log("handleUpdate wird ausgeführt", updatedProject);
```

Comment:
Our coaches are telling us to delete all console.logs...

Reply:
Nice catch!